code/game/machinery/doors/poddoor.dm

```diff
@@ -16,56 +15,70 @@
 	damage_deflection = 70
 	var/datum/crafting_recipe/recipe_type = /datum/crafting_recipe/blast_doors
 	var/deconstruction = BLASTDOOR_FINISHED // deconstruction step
+	var/id = 1
```

Comment:
since these are vars, dont they need /// ?

Reply:
i didn't add this, also i think a comment is only really necessary for vars that _need_ explaining